code/modules/materials/material_sheets.dm

```diff
@@ -42,6 +42,9 @@
 /obj/item/stack/material/get_material()
 	return material
 
+/obj/item/stack/material/proc/get_default_type()
+	return default_type
+
```

Comment:
getters are bad and should be avoided and this is a bad one

Reply:
Aye but the code goes cjkjrsh without it, unfortunately.